js/lib/renderjson.js

```diff
@@ -51,7 +51,7 @@
 //     .array.syntax  ("[", "]")
 
 var module;
-(module||{}).exports = renderjson = (function() {
+(module||{}).exports = window.renderjson = (function() {
```

Comment:
Should we add a comment somewhere that this has been modified so we don't overwrite it by updating the library?